### DISCUSSION TITLE
Temporary failure in name resolution

### DISCUSSION THREAD

**Discussion by d1s2r4a** [YOURLS/YOURLS#2941] [Q&A] [answered]
* [x] I've read the [Troubleshooting First Steps](https://github.com/YOURLS/YOURLS/wiki/Troubleshooting-first-steps)
* [x] This request isn't a duplicate of an existing issue, opened or closed
* [x] I've read the [docs](https://yourls.org/) and followed them (if applicable)
* [x] This is not a personal support request, that should be posted on [discussions community](https://github.com/YOURLS/YOURLS/discussions)

**Describe the bug**
When I open the admin page of my yourls page (admin/index.php) it gives this error:

Incorrect DB config, or could not connect to DB
PDOException: SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo failed: Temporary failure in name resolution

**To Reproduce**
0. Install YOURLS
1. Go to the admin page
2. See error

**Expected behavior**
It should open the admin page like normal.

**Actual behavior**
It gives the error above.

**Screenshots**
![bug](https://user-images.githubusercontent.com/76944256/118170379-adab9600-b432-11eb-9a14-61b88c578a07.jpeg)

**Versions**
YOURLS 1.8.1
Operating System: Hosted on InfinityFree
Plugins or Themes: None

**Additional context**
My private/public setting is set to private (only admin panel).

**Comment by dgw** [comment, marked as answer]
This _is_ a support question, not a bug report. Similar issues with InfinityFree databases are relatively easy to find via Google search, e.g. https://forum.infinityfree.net/t/pdoexception-sqlstate-hy000-2002-php-network-getaddresses-getaddrinfo-failed/9111

Contact InfinityFree support.

**Reply by d1s2r4a** [reply]
Thanks!
